fieldservice_account_analytic/models/__init__.py

```diff
@@ -0,0 +1,10 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import (
+    account_invoice_line,
```

Comment:
```suggestion
    account_move,
```
Rename to match Odoo - account.move.line is in account_move.py